onboarding_auth/onboardingAuthDomain/src/main/java/com/alexafit/onboardingauthdomain/useCase/LoginUser.kt

```diff
@@ -0,0 +1,21 @@
+package com.alexafit.onboardingauthdomain.useCase
+
+import com.alexafit.onboardingauthdata.model.remote.Login
+import com.alexafit.onboardingauthdata.model.remote.LoginResponse
+import com.alexafit.onboardingauthdata.repository.OnboardingAuthRepository
+
+class LoginUser(
```

Comment:
Is this a usecase?
If so, its a common practice to add "UseCase" at the end of the class

Ex
LoginUseCase